budgetportal/models.py

```diff
@@ -1809,6 +1808,67 @@ def _build_complete_expenditure(cls, records):
         return complete_expenditure
 
 
+class Language(models.Model):
+    name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+
+prov_keys = prov_abbrev.keys()
+prov_choices = tuple([(prov_key, prov_key) for prov_key in prov_keys])
+
+
+class Event(models.Model):
```

Comment:
Sort order is important for events - the date field used for presentation has needed to be free-text. Let's add a date field called `start_date` which we use to sort by. Don't worry about replacing `status` with a current-date check right now.